Tests/CMakeLists.txt

```diff
@@ -113,6 +113,14 @@ function(add_test_rf TEST_NAME TEST_EXE_DIR)
     set_tests_properties(${TEST_NAME} PROPERTIES WILL_FAIL TRUE)
 endfunction(add_test_rf)
 
+# Regression tests no FPE trapping ever
```

Comment:
This test won't have these options `amr.plot_file=plt amr.checkpoint_files_output=0 amr.plot_files_output=1 amrex.the_arena_is_managed=0`, which I think we would want to enforce.

Reply:
Actually I'm confused. It just has the same options as the other tests?

Reply:
You could leave the `RUNTIME_OPTIONS` as is and just add `amrex.signal_handling=0` to this test category.